### ISSUE TITLE
about  PatchTST

### ISSUE THREAD

**Issue by MythosAd** [thuml/Time-Series-Library#30]
According to the paper, PatchTsT have a  self-supervise  pre training procedure. Can your team  will add it soon ? Even if not, thank you very much! !!!

**Comment by MythosAd** [comment]
Another problem  , the MICN script have a error :    lable-length is 96  in origin paper's code。

**Comment by MythosAd** [comment]
Oh, I have a question, I find  that origin ECL datasets are txt-file  and  have't no col-name & date 。For example electricity.txt 
https://github.com/laiguokun/multivariate-time-series-data/tree/master/electricity。  how you add a 'date' col?

**Comment by wuhaixu2016** [comment]
Yes, we add the date col according to its time interval.

**Comment by wuhaixu2016** [comment]
About PatchTST, we do plan to add the pretraining feature into this library. Maybe in this summer.
About MICN, label-len=48 can also obtain a fairly good performance.
